.travis.yml

```diff
@@ -1,9 +1,8 @@
 language: node_js
 node_js:
-  - "6"
-  - "8"
   - "10"
-  - "11"
```

Comment:
Would you mind splitting the droppage of older Node versions into a separate PR? (will also need to include the update to `package.json`'s `engines` field in that one too)

Reply:
Done. Moved this change to another [PR](https://github.com/ember-cli/broccoli-uglify-sourcemap/pull/241).

Reply:
But here the `terser.minify()` is async by default from `v5.0.0-beta.0`. But in this package, `async` behavior was optional and configured by the host application. 

This behavior has introduced the following questions
- Will this new change deprecates the `async` property (config)? 
- Also, what would be the behavior of the case where the `async` prop was explicitly set to `false` by the user?

Reply:
As discussed, I have deprecated the `async` support and thrown an [error](https://github.com/ember-cli/broccoli-uglify-sourcemap/pull/237/files#diff-168726dbe96b3ce427e7fedce31bb0bcR48) if the `async` property is passed inside inside `options` object.